lib/data/external/GcpClient.js

```diff
@@ -28,17 +37,18 @@ class GcpClient extends AwsClient {
         this.type = 'GCP';
         this._gcpBucketName = config.bucketName;
         this._mpuBucketName = config.mpuBucket;
-        this._overflowBucketName = config.overflowBucket;
+        this._createGcpKey = this._createAwsKey.bind(this);
```

Comment:
Do you really need a bind here?

Reply:
It doesn't. the helper function does not use any instance properties. will fix